examples/gallery/images/image.py

```diff
@@ -1,24 +1,23 @@
 """
-Images or EPS files on maps
----------------------------
-The :meth:`pygmt.Figure.image` method can be used to read and
-place a raster image file or an Encapsulated PostScript file
-on a map. We must specify the file as *str* via the ``imagefile``
-parameter or simply use the filename as the first argument. You can
-also use a full URL pointing to your desired image. The ``position``
-parameter allows us to set a reference point on the map for the image.
-
+Images on figures
+-----------------
+The :meth:`pygmt.Figure.image` method can be used to read and place an image
+file in many formats (e.g., png, jpg, eps, pdf) on a figure. We must specify
+the file as *str* via the ``imagefile`` parameter or simply use the filename as
+the first argument. You can also use a full URL pointing to your desired image.
+The ``position`` parameter allows us to set a reference point on the map for
+the image.
 """
 import os
 
 import pygmt
 
 fig = pygmt.Figure()
-
 fig.basemap(region=[0, 2, 0, 2], projection="X6c", frame=True)
```

Comment:
The example may look better for a larger image size.
```suggestion
fig.basemap(region=[0, 2, 0, 2], projection="X10c", frame=True)
```